calcom/cal.com
# Updating contributing to app store docs

## What does this PR do?

Updating the doc "Contributing to App Store" doc with @zomars's changes. 


## Type of change




## How should this be tested?



- Visit http://localhost:4000/developer/app-store
- Ensure that `yarn docs-build` runs without errors
